db/migrate/20160422114747_copy_community_id_from_people_to_emails.rb

```diff
@@ -0,0 +1,23 @@
+class CopyCommunityIdFromPeopleToEmails < ActiveRecord::Migration
+  def up
+    exec_update(
+      ["UPDATE emails e",
+       "LEFT JOIN people p ON p.id = e.person_id",
+       "SET e.community_id = -1",
+       "WHERE p.community_id = -1"].join(" "),
+      "Set community_id -1",
+      []
+    )
+  end
```

Comment:
I was thinking that this is also a good place to clean up the `emails` table. If there are any `emails` to do not match to any `person`, those emails can be removed. So after this migration is run, we can have a look at the DB if there are any orphan emails (`community_id` still `NULL`) and remove them.